site/en/r2/tutorials/load_data/tf_records.ipynb

```diff
@@ -1067,7 +1053,7 @@
         "id": "Azx83ryQEU6T"
       },
       "source": [
-        "As we did earlier, we can now encode the features as types compatible with `tf.Example`. In this case, we will not only store the raw image string as a feature, but we will store the height, width, depth, and an arbitrary `label` feature, which is used when we write the file to distinguish between the cat image and the bridge image. We will use `0` for the cat image, and `1` for the bridge image."
+        "As you did earlier, you can now encode the features as types compatible with `tf.Example`. In this case, you will store the raw image string feature, as well as the height, width, depth, and arbitrary `label` feature. The latter is used when you write the file to distinguish between the cat image and the bridge image. Use `0` for the cat image, and `1` for the bridge image:"
```

Comment:
Not your fault, but I'd remove you/we where possible and tell the reader what to do as simply as possible (there's kind of an implied "you" once established earlier in the page). Something like:
"As before, encode the features ..."
"This stores the raw ..."

Reply:
Thanks @lamberta , appreciate it. I went through the notebook once more and replaced the "you + [take action]" with instruction-like "[Take action]" where I thought it was more applicable. The [Google docs style guide](https://developers.google.com/style/person) also helped. Cheers 👍